pkg/build/controller/build/overrides/overrides_test.go

```diff
@@ -357,3 +358,129 @@ func TestBuildOverrideAnnotations(t *testing.T) {
 		}
 	}
 }
+
+func TestBuildOverrideTolerations(t *testing.T) {
+	tests := []struct {
+		buildTolerations    []corev1.Toleration
```

Comment:
nit - for table-driven tests:

1. Include a `name string` field. 
2. Run inside a separate test run:

```
for _, tc := range tests {
  t.Run(tc.name, func(t *testing.T) {
      ....
  })
}
```

This makes it easy to identify test cases that fail, and it _may_ also let the tests run in parallel.

Reply:
updated in next push